servers/audio_server.cpp

```diff
@@ -65,19 +65,28 @@ void AudioDriver::audio_server_process(int p_frames, int32_t *p_buffer, bool p_u
 }
 
 void AudioDriver::update_mix_time(int p_frames) {
+	lock();
```

Comment:
This is called from `audio_server_process()`, where the lock is already held, so I think it's better not trying to relock here. Should be fine, provided the mutex is recursive, but anyway.

Reply:
Oh neat, so I don't actually have to lock in update_mix_time, just in the two `get_*` methods. Easy fix.